google/cloud/forseti/services/client.py

```diff
@@ -277,14 +278,15 @@ def query_access_by_permissions(self,
 
         Returns:
             object: Generator yielding access tuples.
+            :param expand_resources:
```

Comment:
This might be a copy/paste mistake, I don't think it is part of the return value.

Reply:
IDE refactoring vestiges. Done!